jxl/src/features/spline.rs

```diff
@@ -639,8 +638,12 @@ impl Splines {
             let spline = qspline.dequantize(
                 &self.starting_points[index],
                 self.quantization_adjustment,
-                y_to_x,
-                y_to_b,
+                color_correlation_params.base_correlation_x
+                    + (color_correlation_params.ytox_lf as f32)
+                        / (color_correlation_params.color_factor as f32),
+                color_correlation_params.base_correlation_b
+                    + (color_correlation_params.ytob_lf as f32)
+                        / (color_correlation_params.color_factor as f32),
```

Comment:
(Or would we prefer to have this as methods of `ColorCorrelationParams`?)

Reply:
I think that would be better